source/support.tex

```diff
@@ -4188,9 +4188,9 @@
 
   // \ref{cmp.concept}, concept \libconcept{three_way_comparable}
   template<class T, class Cat = partial_ordering>
-    concept three_way_comparable = @\seebelow@;
+    concept @\deflibconcept{three_way_comparable}@ = @\seebelow@;
   template<class T, class U, class Cat = partial_ordering>
-    concept three_way_comparable_with = @\seebelow@;
+    concept @\deflibconcept{three_way_comparable_with}@ = @\seebelow@;
```

Comment:
This should not be `\deflibconcept` here; the `\deflibconept` for these is further down, with the actual (semantic) definition. Make it a `\libconcept`, please.

Reply:
right - I caught that also and fixed.  You're too fast for me! :)

Reply:
Oh, and as far as making it a \libconcept, we don't seem to do that in general for the declaration.  Should we? In the synopsis, we (typically) only use \libconcept in requires clauses and parameters. We do use \exposconcept for the \expos case however (which makes sense).  See:
`grep "^[ ]*concept .*=" *.tex |grep -v deflibcon |grep -v defexpos |grep -v seebelow`

Reply:
It's not really useful to add "use" index entries pointing to the synopsis of a definition if we have a proper `\itemdecl` definition in the vicinity.

Reply:
agreed